trio/_tests/test_exports.py

```diff
@@ -492,4 +492,8 @@ def test_classes_are_final():
                 continue
             # ... insert other special cases here ...
 
+            # don't care about the *Statistics classes
```

Comment:
Instead of opting these out, wouldn't it be better to make them final? People shouldn't be inheriting from them.

Reply:
I don't really see any strong reason to *forbid* people from inheriting them either? I could plausibly see somebody writing a wrapper around one of the locks and them deeming it useful to inherit from the Statistics class.